server/main_server/src/main.cpp

```diff
@@ -280,11 +274,17 @@ auto main(int _argc, char* _argv[]) -> int
         log_server::info("{}: Initializing shared memory for main server process.", __func__);
 
         namespace hnc = irods::experimental::net::hostname_cache;
```

Comment:
interested in pulling these out of the experimental namespace?

also still noticed 'log' is in experimental too...

Reply:
Yes, but I'm afraid we may run out of time to complete that effort since the blast radius is wide. I'm mostly speaking about the logger. The hostname and dns cache are pretty simple to handle.

I'll open issues for that effort so we can circle back to them after everything for iRODS 5 is confirmed to be working.

Reply:
Created #8399, #8400, #8403, and a few others for various libs.